JsonSchema/JsonSerializerOptionsExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+
+namespace Json.Schema;
+
+// TODO: .NET 5+ would have these methods marked with `RequiresUnreferencedCodeAttribute` to warn against tree trimming
+internal static class JsonSerializerOptionsExtensions
```

Comment:
I created Json.More specifically to hold more general-use extensions to STJ that I think probably should have been included.  This seems to fit.

I'm happy with leaving the `object? Read()` method here since it does seem to solve this specific problem of not knowing what we're going to get back.

Reply:
I already marked this ready for review, but I'm resolving this comment in a commit right now, sorry for the early notification.

Reply:
Generally, I prefer the commenter to resolve their own comment to assure that their concern has been addressed.